.github/r_code/pdflatex_local_manual.r

```diff
@@ -0,0 +1,33 @@
+#' Comment out lines that prevent pdf from being generated locally and produce pdf
+#' @description Reads the tex file and comments out lines that prevent pdf from being
+#'   generated locally and  produces pdf using tinytex::pdflatex() and then changes 
+#'   the tex file back.
+#' @param manual_file Name of the manual tex file. Default is "SS330_User_Manual.tex"
```

Comment:
It is not clear from the description that you cannot put the full directory to the path in the parameter or that it is assumed that the file is present in your working directory. This is called a code side effect, which is frowned upon unless declared.

Reply:
Okay, yes my assumption is that this function is only for working within the github repo on your local machine where the file is in the working directory. I added a dir param if this is not always the case.